src/app/(proper_react)/DashboardTopBanner.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "../tokens";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  border: 1px solid rgba($color-purple-50, 0.2);
+  border-radius: $border-radius-md;
+  padding: $layout-lg $spacing-xl;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-sm;
+
+    h3 {
+      font: $text-title-2xs;
+    }
+
+    .cta {
+      margin-top: $spacing-sm;
+      align-self: start;
+    }
+  }
+
+  & > * {
+    flex: 1;
+  }
+
+  &:before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
```

Comment:
I think it's because of this, but the entire top banner is unclickable for me. Any reason this couldn't be a background image on the container?